### PR TITLE
Nerfs the movement grab-breaking a bunch.

### PR DESCRIPTION
**Pull request by Useroth**
:cl:  
tweak: Simply breaking out of grabs by walking away is now harder. Resist button's effectiveness remains the same.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -637,13 +637,14 @@
 /mob/living/resist_grab(moving_resist)
 	. = 1
 	if(pulledby.grab_state)
-		if(!resting && prob(30/pulledby.grab_state))
+		var/resist_chance = (moving_resist == TRUE) ? 10/pulledby.grab_state : 30/pulledby.grab_state
```
Comment (deathride58) [inline code/modules/mob/living/living.dm]: This var is only used one single time. Defining a var that only gets used one single time is a pretty bad coding practice in the context of byond, as declaring a var has a pretty hefty performance overhead at runtime.
Move the contents of this var into the below prob() call

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (Useroth) [reply code/modules/mob/living/living.dm]: Done.

**Comment by silicons** [comment]
ok but why do grabs need a buff in their current state?

**Comment by Useroth** [comment]
I mean, it merely raises the skill bar a little bit, so that breaking grabs isn't a completely brainless mechanic where you can rather easily simply walk away from an aggro-grab without even thinking about it. The functionality of mashing RESIST is entirely unaffected by this PR.

**Comment by silicons** [comment]
eh nah maybe after grab rework aggro grabs instantly stun you with no recourse other than using rng it doesn't need buffing at all.

**Pull request closed without merging**